pkg/southbound/mocks/target_mock.go

```diff
@@ -0,0 +1,112 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
```

Comment:
Can we rename this file to target_mock_test.go? If we do that, the coverage tools will not count the mocked code in here against us.

Reply:
Actually I can't - if I make this _test.go then the main code can't find it.
Also the linter complains I have a test file but nothing to test it on. I'd have to create a source file with dummy code.